src/main.rs

```diff
@@ -916,12 +918,20 @@ where
             }
             RecordIdState::UnknownRemoteId(remote_id, missing) => {
                 missing.commit()?;
-                error!("Identifier not in database: '{remote_id}'");
+                if !ignore_errors {
+                    error!("Identifier not in database: '{remote_id}'");
+                }
             }
             RecordIdState::UndefinedAlias(alias) => {
-                error!("Undefined alias: '{alias}'");
+                if !ignore_errors {
+                    error!("Undefined alias: '{alias}'");
+                }
+            }
+            RecordIdState::InvalidRemoteId(err) => {
+                if !ignore_errors {
+                    error!("{err}")
+                }
```

Comment:
Should we emit this error anyway? For the other two, it is natural that they are suppressed.

Reply:
This is the error if the Remote Id is invalid

Reply:
I think it's fine to suppress it, but I'm happy either way.